operatorcourier/courier_api.py

```diff
@@ -0,0 +1,36 @@
+"""
+operatorcourier.courier_api module
+
+This module implements the api that should be imported when using the operator-courier package
+"""
+
+from operatorcourier.build import BuildCmd
+from operatorcourier.validate import ValidateCmd
+
+def build_and_verify(source_dir="", files=""):
```

Comment:
Is better to use `None` as default vaules instead of empty string

Reply:
Addressed in latest commit.